static/app/components/feedback/feedbackItem/feedbackActions.tsx

```diff
@@ -48,11 +48,22 @@ export default function FeedbackActions({
 }
 
 function LargeWidth({feedbackItem}: {feedbackItem: FeedbackIssue}) {
-  const {isResolved, onResolveClick, isSpam, onSpamClick, hasSeen, onMarkAsReadClick} =
-    useFeedbackActions({feedbackItem});
+  const {
+    hasDelete,
+    onDelete,
+    isResolved,
+    onResolveClick,
+    isSpam,
+    onSpamClick,
+    hasSeen,
+    onMarkAsReadClick,
+  } = useFeedbackActions({feedbackItem});
 
   return (
     <Fragment>
+      <Button size="xs" onClick={onDelete} priority="danger" disabled={!hasDelete}>
```

Comment:
ooh, we should split up the two conditions the feed into this:

```
const hasDelete =
    organization.access.includes('event:admin') &&
    organization.features.includes('issue-platform-deletion-ui');
```

if the `organization.access` isn't good enough, the i think disabled makes sense..

but if the `organization.features` is not set then we can probably hide the button.